packages/diagrams/backend/sirius-components-diagrams-layout/src/main/java/org/eclipse/sirius/components/diagrams/layout/LayoutConfiguratorRegistry.java

```diff
@@ -92,33 +94,34 @@ public ISiriusWebLayoutConfigurator getDefaultLayoutConfigurator() {
                 .setProperty(LayeredOptions.SPACING_EDGE_NODE_BETWEEN_LAYERS, SPACING_NODE_EDGE);
 
         configurator.configureByType(NodeType.NODE_RECTANGLE)
```

Comment:
Should not we have the same config for `NodeType.NODE_RECTANGLE` and `NodeType.NODE_IMAGE`?
diff= NODE_SIZE_CONSTRAINTS and NODE_SIZE_OPTIONS